snappy-core/src/main/scala/org/apache/spark/sql/SnappyContext.scala

```diff
@@ -322,20 +328,21 @@ class SnappyContext protected[spark](@transient override val sparkContext: Spark
   }
 
   /**
-   * Creates a Snappy managed table. Any relation providers (e.g. parquet, jdbc etc)
-   * supported by Spark & Snappy can be created here. Unlike SqlContext.createExternalTable this
-   * API creates a persistent catalog entry.
-   *
-   * {{{
-   *
-   * val airlineDF = snappyContext.createTable(stagingAirline, "parquet", Map("path" -> airlinefilePath))
-   *
-   * }}}
-   * @param tableName Name of the table
-   * @param provider  Provider name such as 'COLUMN', 'ROW', 'JDBC', 'PARQUET' etc.
-   * @param options Properties for table creation
-   * @return DataFrame for the table
-   */
+    * Creates a Snappy managed table. Any relation providers (e.g. parquet, jdbc etc)
+    * supported by Spark & Snappy can be created here. Unlike SqlContext.createExternalTable this
+    * API creates a persistent catalog entry.
+    * (Scala-specific)
+    * {{{
+    *
+    * val DF = snappyContext.createTable(stagingAirline, "parquet", Map("path" -> airlinefilePath))
+    *
+    * }}}
+    *
+    * @param tableName Name of the table
+    * @param provider  Provider name such as 'COLUMN', 'ROW', 'JDBC', 'PARQUET' etc.
+    * @param options   Properties for table creation
+    * @return DataFrame for the table
+    */
   def createTable(
```

Comment:
Please revert all these formatting changes. I believe current formatting of these is proper as per Spark guidelines that we have been following. Please ensure the application of codeStyleSettings.xml (as in docs/build-instructions.md).